contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,669 @@
+  Future use cases include dynamically provisioning and intelligent discovery of existing devices, which this proposal sets the 
+  foundation for more fully developing these methods. 
+
+ 
+# Design Overview
+
+  The proposed design is based on the idea of leveraging well defined concepts for storage in Kubernetes. The consumption and 
+  definitions for the block devices will be driven through the PVC and PV definitions. Along with Storage
+  Resource definitions, this will provide the admin with a consistent way of managing all storage. 
+  The API changes proposed in the following section are minimal with the idea of defining a volumeMode to indicate both the definition
+  and consumption of the devices. Since it's possible to create a volume as a block device and then later consume it by provisioning
+  a filesystem on top, the design requires explicit intent for how the volume will be used.
+  The additional benefit of explicitly defining how the volume is to be consumed will provide a means for indicating the method
+  by which the device should be scrubbed when the claim is deleted, as this method will differ from a raw block device compared to a 
+  filesystem. The ownership and responsibility of defining the retention policy shall be up to the plugin method being utilized and is
+  not covered in this proposal.
+  
+  Limiting use of the volumeMode to block can be executed through the use of storage resource quotas and storageClasses defined by the 
+  administrator.
+  
+  To ensure backwards compatibility and a phased transition of this feature, the consensus from the community is to intentionally disable
+  the volumeMode: Block for external provisioners until a suitable implementation for provisioner versioning has been accepted and 
+  implemented in the community. In addition, in-tree provisioners should be able to gracefully ignore volumeMode API objects for plugins
+  that haven't been updated to accept this value.
+  
+  It is important to note that when a PV is bound, it is either bound as a raw block device or formatted with a filesystem. Therefore, 
+  the PVC drives the request and intended usage of the device by specifying the volumeMode as part of the API. This design lends itself
+  to future support of dynamic provisioning by also letting the request initiate from the PVC defining the role for the PV. It also 
+  allows flexibility in the implementation and storage plugins to determine their support of this feature. Acceptable values for 
+  volumeMode are 'Block' and 'Filesystem'. Where 'Filesystem' is the default value today and not required to be set in the PV/PVC.
+  
+# Proposed API Changes
+   
+## Persistent Volume Claim API Changes:
+In the simplest case of static provisioning, a user asks for a volumeMode of block. The binder will only bind to a PV defined 
+with the same volumeMode.
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+name: myclaim
```

Comment:
this should be indented 2 spaces or you get an error 

```
error validating "local-pv.yaml": error validating data: ValidationError(PersistentVolume): unknown field "name"
```

same with pvc definition

Reply:
@screeley44 done, thanks